.werft/jobs/build/job-config.ts

```diff
@@ -0,0 +1,161 @@
+import { config } from "process";
+import { exec } from "../../util/shell";
+import { Werft } from "../../util/werft";
+
+export interface JobConfig {
```

Comment:
Super nice to see the usage of interfaces! Our CI can finally look like a proper software :)